### ISSUE TITLE
1.19.2 in servers texturepack loading bug

### ISSUE THREAD

**Issue by Catfl** [kennytv/kennytvs-epic-force-close-loading-screen-mod-for-fabric#17]
in servers went the servers auto loading texture pack keyboard don't work on login

**Comment by kennytv** [comment]
Can you rephrase? I don't understand the issue

**Comment by Catfl** [comment]
Sorry, I didn’t see the error report clearly. In fact, there is no problem. It is caused by a mod that opens the accelerated startup interface.

**Comment by Catfl** [comment]
I asked the same mod author before, and he found something wrong. I checked the problem again and found that the mod that speeds up the pop-up window of minecraft caused the problem.
